docs/output/targeted.md

```diff
@@ -0,0 +1,238 @@
+---
+order: 1
+---
+
+# nf-core/crisprseq: Output
+
+## Introduction
+
+This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
```

Comment:
Should this not be more specific about the targeted analysis? Like the `screening.md`?